packages/trader/src/Modules/CFD/Containers/compare-accounts-content.tsx

```diff
@@ -0,0 +1,690 @@
+import React from 'react';
+import { Table, Text, ThemedScrollbars, Div100vhContainer } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { isDesktop, CFD_PLATFORMS, isLandingCompanyEnabled, PlatformContext } from '@deriv/shared';
+import { LandingCompany } from '@deriv/api-types';
+
+type TCFDAttributeDescriberProps = {
+    name: string;
+    counter: number | null;
+};
+
+type TFilterAvailableAccounts = (
+    landing_companies: LandingCompany,
+    table: TAccountsDescription[],
+    is_logged_in: boolean,
+    show_eu_related: boolean,
+    platform: string,
+    is_australian: boolean
+) => Array<{ [key: string]: string | React.ReactNode | undefined }>;
+
+type TAccountsDescription = {
+    attribute: string;
+    mt5: TDxTradeAccountsDescription & { financial_stp: string };
+    dxtrade: TDxTradeAccountsDescription;
+};
+
+type TDxTradeAccountsDescription = {
+    synthetic: string;
+    synthetic_eu: string;
+    financial: string;
+    financial_au: string;
```

Comment:
There is no need of semicolon(;) or comma(,) symbols in typescript

Reply:
Adding semicolon or comma are both valid in ts.. You can refer here

https://medium.com/swlh/typescript-best-practices-delimiters-and-ordering-7feb42e0ca1d#:~:text=In%20TypeScript%2C%20interfaces%20and%20type,to%20have%20a%20consistent%20style.&text=They're%20both%20valid%2C%20but%20TypeScript%20prefers%20semicolon.